github/repos_collaborators.go

```diff
@@ -26,6 +26,18 @@ type ListCollaboratorsOptions struct {
 	ListOptions
 }
 
+//AddCollaboratorResponse https://developer.github.com/v3/repos/collaborators/#response-when-a-new-invitation-is-created
```

Comment:
Let's please call this a `CollaboratorInvitation` since that is how the GitHub docs refer to it.
Also, this needs to be a complete sentence and formatted similarly to other ones in the repo.  For example:

```
// CollaboratorInvitation represents an invitation created when adding a collaborator.
// GitHub API docs: https://developer.github.com/v3/repos/collaborators/#response-when-a-new-invitation-is-created